courses/api/pagination.py

```diff
@@ -0,0 +1,10 @@
+from rest_framework.pagination import (
+    PageNumberPagination
+)
+
+
```

Comment:
Why not have a custom class which inherits page number pagination and custom pages limit params. In the Viewset you call this page class and pass a constant specific to that vs. You also keep a default value in case page limit is not passed